File: 1929-concatenation-of-array/1929-concatenation-of-array.go
func getConcatenation(nums []int) []int {
    
//     var ans []int
    
//     for _, v := range nums {
//         ans = append(ans,v)
//     }
    
//     if len(ans) == len(nums) {
//         for _, v := range nums {
//             ans = append(ans,v)
//         }
//     }
    
//     return ans
// }
    
    ans := make([]int, len(nums))
    
    copy(ans, nums)
    
    ans = append(ans,nums...)
    return ans
}

File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        
        ans = []
        
        for i in nums:
            ans.append(i)
        
        if len(ans) == len(nums):
            for i in nums:
                ans.append(i)
        return ans
        